src/Seq.App.Slack/SlackApi.cs

```diff
@@ -5,7 +5,12 @@
 
 namespace Seq.App.Slack
 {
-    class SlackApi
+    public interface ISlackApi
```

Comment:
In this project I think the convention is to move this to its own `ISlackApi.cs` file.

Reply:
Good to know, I've moved this to it's own file